front/views/settings/SettingsProfileView.tsx

```diff
@@ -55,6 +55,16 @@ const ProfileSettings = ({ navigation }: { navigation: any }) => {
 								},
 							},
 						},
+						{
+							type: 'text',
+							title: translate('verified'),
+							data: {
+								text: user.emailVerified ? 'verified' : 'not verified',
```

Comment:
on est d'accord ça va faire Vérifié -> verified/not verified sur la même ligne mdr